src/lib/npm-client.ts

```diff
@@ -155,20 +154,27 @@ export class NpmPublishClient {
             const body = createTgz(publishedDirectory, reject);
             const metadata = { readme, ...packageJson };
             if (dry) {
-                log("(dry) Skip publish of " + publishedDirectory);
+                log(`(dry) Skip publish of ${publishedDirectory} to ${this.registry}`);
             }
             resolve(dry ? undefined : promisifyVoid(cb => {
-                this.client.publish(npmRegistry, { access: "public", auth: this.auth, metadata, body }, cb);
+                this.client.publish(this.registry, { access: "public", auth: this.auth, metadata, body }, cb);
             }));
         });
     }
 
-    tag(packageName: string, version: string, tag: string): Promise<void> {
-        return promisifyVoid(cb => { this.client.tag(packageUrl(packageName), { version, tag, auth: this.auth }, cb); });
+    tag(packageName: string, version: string, distTag: string, dry: boolean, log: Logger): Promise<void> {
+        return promisifyVoid(cb => {
+            if (dry) {
+                log(`(dry) Skip tag of ${packageName}@${distTag} as ${version}`);
+                cb(undefined);
```

Comment:
Is there a less clunky way to write this? I haven't written much promisified code, so I fell back to my base knowledge of the monads+continuations. (It also made me want to rename `cb` to `k` or `return_`, but I refrained.)

Reply:
I’m not familiar with `promisifyVoid`, but you could move `if (dry) { ... }` outside the closure and `return Promise.resolve()` instead of `cb(undefined)`?

Reply:
It's a utility function lower in the file, but it doesn't seem that useful.

I'll use `return Promise.resolve()`.